spec/clearance/session_spec.rb

```diff
@@ -399,6 +445,15 @@ def stub_status(success)
     end
   end
 
+  # a bit of a hack to get the cookies that ActionDispatch sets inside session
+  def remember_token_cookie(session, cookie_name = "remember_token")
```

Comment:
this looks like a false positive, because this param is used in the eval block ... I had to refactor the `instance_eval` method on e6864d0eacd226617ecf1a9e48ff3447733afe14 to fit with another styleguide warning, so I guess you have to pick between those two?